.golangci.yml

```diff
@@ -0,0 +1,10 @@
+run:
+    skip-files:
+        - ".*_test\\.go$"
+
+linters-settings:
+    golint:
+        min-confidence: 0.1
+
+linters:
+    # enable-all: true
```

Comment:
FYI: enable-all is deprecated. With the last couple release golangci-lint received a few linters that makes the blocklist even longer, so allowlist as a default strategy is recommended: https://golangci-lint.run/usage/configuration/